src/Providers/LaunchDarkly/LaunchDarklyProvider.php

```diff
@@ -58,10 +58,16 @@ public function setAnonymousUser(): void
 
     public function flag(string $flag): bool
     {
-        if (!$this->client) {
+        $client = $this->client;
```

Comment:
This is to fix a phpstan errors that were not aware of the check below and was saying that the variation method is not available.